web/site/content/en-CA/products/br/overview.md

```diff
@@ -138,4 +138,8 @@ Updates of note to this page are recorded here.
     <td>2025-04-04</td>
     <td>Update Sandbox Postman collection</td>
   </tr>
+  <tr>
+    <td>2025-04-23</td>
+    <td>Update Sandbox Postman collection</td>
+  </tr>
```

Comment:
can we also add a note to this page history entry? maybe something about affiliation/delegation examples added